repos/application-insights.proj

```diff
@@ -1,6 +1,12 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup>
+    <PreferredHash>53b80940842204f78708a538628288ff5d741a1d</PreferredHash>
+    <SourceDirectory>ApplicationInsights-dotnet</SourceDirectory>
```

Comment:
Is this directory dictated by something else?  Is there a reason that it's changing from what we had before?

Reply:
All of the source directories now use the same name as the repos being cloned.  This ended up being hard to work around in the uberclone and easier to work around for us, so I went in this direction.  We could also rename the `repo.proj`s if you think that would be better - I'm fine with either.